kinematics_node:
  ros__parameters:
    wheel_separation: 0.451
    wheel_radius: 0.108
